src/main/java/uk/gov/hmcts/reform/sscs/service/placeholders/FurtherEvidencePlaceholderService.java

```diff
@@ -94,15 +96,37 @@ private String extractNameJointParty(SscsCaseData caseData) {
             .orElse("Sir/Madam");
     }
 
+    private String getNameOtherPartyAddress(SscsCaseData caseData, String otherPartyId) {
```

Comment:
Can't you call getOtherPartyAddress method and extract the name from the returning address?

Reply:
The address and name aren't the same object so not sure what you mean here? The method name (getNameOtherPartyAddress()) doesn't actually make sense so I have updated it to be clearer - hopefully it makes sense now?